scripts/source/01-make_bem_surfaces.py

```diff
@@ -0,0 +1,85 @@
+"""
+===================
+Create BEM surfaces
+===================
+
+Extract the BEM surfaces using the watershed algorithm. This is required to
+produce the forward solution in the next step.
+"""
+
+import logging
+from pathlib import Path
+
+import mne
+from mne.parallel import parallel_func
+
+import config
+from config import gen_log_message, on_error, failsafe_run
+
+logger = logging.getLogger('mne-study-template')
+
+
+@failsafe_run(on_error=on_error)
+def make_bem(subject):
+    fs_subject = config.get_fs_subject(subject)
+    fs_subjects_dir = config.get_fs_subjects_dir()
+    mri_dir = Path(fs_subjects_dir) / fs_subject / 'mri'
+    bem_dir = Path(fs_subjects_dir) / fs_subject / 'bem'
+    watershed_bem_dir = bem_dir / 'watershed'
+    flash_bem_dir = bem_dir / 'flash'
+    flash_dir = mri_dir / 'flash' / 'parameter_maps'
```

Comment:
hum technically we should write the flash images in the BIDS dataset in the anat folder and use these. As written it can only work for the sample dataset.